modules/apps/27-interchain-accounts/controller/client/cli/tx.go

```diff
@@ -0,0 +1,62 @@
+package cli
+
+import (
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/ibc-go/v5/modules/apps/27-interchain-accounts/controller/types"
+)
+
+const (
+	// The controller chain channel version
+	flagVersion = "version"
+)
+
+// NewTxCmd creates and returns the tx command
+func NewTxCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:                        "controller",
+		Short:                      "ica controller transactions subcommands",
+		DisableFlagParsing:         true,
+		SuggestionsMinimumDistance: 2,
+		RunE:                       client.ValidateCmd,
+	}
+
+	cmd.AddCommand(
+		newRegisterAccountCmd(),
+	)
+
+	return cmd
+}
+
+func newRegisterAccountCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "register [connection-id]",
+		Short: "Register account via connection end identifier on the controller chain",
+		Args:  cobra.ExactArgs(1),
```

Comment:
Might be good to add a `Long` section which explains that the connection identifier should be for the source chain and that the account will be created on the counterparty chain

Reply:
Could you see and make sure if the `Long` description is ok?